docs/ethical-advertising.rst

```diff
@@ -4,13 +4,26 @@ Ethical Advertising
 Read the Docs is a large,
 free web service.
 There is one proven business model to support this kind of site:
-Advertising.
-We're building an advertising model that respects users while providing value to advertisers.
+**Advertising**.
+I know it's a dirty word,
```

Comment:
With this lede/lead you're starting from a mindset... it's like an elementary school janitor showing up to work each morning and saying "all kids are dirty, chaotic, and they get sick all the time" when really, the reason why the elementary school janitor shows up to work every day is to make the school a healthy, clean environment for students to learn. Okay, this example might be over-the-top emotionally, but I think there's a similar vein of truth - we want to keep the site going and maintain the level of quality expected but not by being associated with something "dirty."

So, can you revise to lead with a more positive outlook perhaps? A couple of ideas (feel free to throw out, of course!)

"Advertising in our context is simply marketing communications to a pretty unique and incredible market - you and your users. In our case, we are giving the market something relevant and delivered through the sites infrastructure itself - targeted ads."

"A sustainable business model needs a market to achieve its goals. Believe it or not, some of the most profitable business models on the Internet are ads, affiliate programs, and drop-shipping items like t-shirt designs. While we would consider many inventive ways to sustain the growth and stability patterns we've established, right now we think that shipping things is less sustainable than shipping bits, so we chose to use ads, shipped on the various content sites, to make enough money to keep this crazy ride going."

You could dive further into sustainability in the business model. I especially like a model from Zingerman's, a deli in Michigan: http://www.zingermanscommunity.com/about-us/culture/sustainability/ Happy to dig into this further than this review if you want!